scripts/common_startup.sh

```diff
@@ -9,6 +9,7 @@ done
 # Conda Python is in use, do not use virtualenv
 if python -V 2>&1 | grep -q -e 'Anaconda' -e 'Continuum Analytics' ; then
     SET_VENV=0
+    CONDA_ALREADY_INSTALLED=1
```

Comment:
Nicola, thanks for taking the time to review this :).

Yes I hesitated, and I first did put in there. But then I saw the verbosity and thought it would be nice to put this as far as possible in the logs to have more chances for people to see it. Plus, I thought that it could be an open way for people to add more stuff to handle the case of Conda being already installed.

If you think it is not useful, and others too, I can totally move it back there and remove the extra variable.